internal/pkg_npm/packager.js

```diff
@@ -186,13 +186,30 @@ function main(args) {
    * @returns {string}
    */
   function getOwningPackageRelativeOutPath(file) {
+    // below isn't indented correctly so that that we can
+    // have a diff of only added lines from the google code
+    // this should help with merge conflicts / rebases and
+    // make this code less likely to accidentally be broken.
+    let relativePath = (() => {
     for (const workspaceName of vendorExternal) {
       if (file.shortPath.startsWith(`../${workspaceName}`)) {
         return path.relative(`../${workspaceName}`, file.shortPath);
       }
     }
 
     return path.relative(owningPackageName, file.shortPath);
+    })();
+    // I changed this from the master branch to slightly misuse the
```

Comment:
This is a little off topic. What about something like:
This function has been modified to additionally strip prefixes off filenames.
This is leveraged by lucid-extension-sdk to maintain compatibility between internal and external typescript targets